src/System Application/App/Performance Profiler/src/PerformanceProfilerDelete.Codeunit.al

```diff
@@ -0,0 +1,43 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Tooling;
+
+using System.DataAdministration;
+
+codeunit 1930 "Shpfy Log Entries Delete"
```

Comment:
you may want to reconsider the name of this codeunit?

Reply:
busy morning... fixed